cli/dcos-spark/submit_builder.go

```diff
@@ -620,6 +689,27 @@ func buildSubmitJson(cmd *SparkCommand) (string, error) {
 		}
 	}
 
+	// TLS configuration
+	if args.keystoreSecretPath != "" {
+		// Make sure passwords are set
+		if args.keystorePassword == "" || args.privateKeyPassword == "" {
+			return "", errors.New("Need to provide keystore password and key password with keystore")
+		} else {
+			args.properties["spark.ssl.keyStorePassword"] = args.keystorePassword
```

Comment:
nit/question: any reason the password configs are set here instead of in `setupTLSArgs?

Reply:
No particular reason. I'll move the setting into `setupTLSArgs` and just leave the validation check here.